docs/releases/v0.11.0.rst

```diff
@@ -108,6 +108,11 @@
     single-threaded. The parameter accepts any value available via the standard
     python library ``codecs``. [#2395]
 
+  -[new] Added parameter ``memory_limit`` which instructs fread to try to limit
+    the amount of memory used when reading the input. This parameter is
+    especially useful when reading files that are larger than the amount of
+    available memory. [#1750]
```

Comment:
@st-pasha what is API if I want to read `csv` into (fully) jay-backed datatable frame?

Reply:
You would first read it via `fread()` and then save into Jay using `DT.to_jay()`.